### ISSUE TITLE
ui.View.present - hide_status_bar

### ISSUE THREAD

**Issue by zrzka** [omz/Pythonista-Issues#380]
In case of `fullscreen` presentation, I'd value a possibility to hide / show status bar. Since all these `setStatusBarHidden:withAnimation:` methods are deprecated ... would be nice.

**Comment by dgelessus** [comment]
Apparently the preferred way to show/hide the status bar is now [via the view controller](https://developer.apple.com/documentation/uikit/uiviewcontroller?language=objc#1654546). Since Pythonista's `ui.View` is kind of equivalent to a view controller (I think?), maybe the status bar visibility and style should be exposed as `ui.View` attributes. This would also allow changing the status bar after the view has been presented.

**Comment by zrzka** [comment]
Yes, it is preferred way and the view controller for presented `ui.View` is `SUIViewController_PY3`. Personally I don't need functions for hiding / showing status bar. For me, it's perfectly okay if I pass `hide_status_bar=True` and `prefersStatusBarHidden` returns `hide_status_bar`. That's all I need.

I can get the view controller via [nextResponder](https://developer.apple.com/documentation/uikit/uiresponder/1621099-nextresponder) (actually it returns `UIViewController` for `UIView`), play some swizzle games, ..., but `hide_status_bar` is preferred :) Docs quote:

> The UIResponder class does not store or set the next responder automatically, so this method returns nil by default. Subclasses must override this method and return an appropriate next responder. For example, UIView implements this method and returns the 
UIViewController object that manages it (if it has one) or its superview (if it doesn’t).
